en/02_Developer_Guides/00_Model/02_Relations.md

```diff
@@ -448,26 +560,35 @@ class Tag extends DataObject
     /**
      * Example iterator placeholder for belongs_many_many.
      * This is a list of arbitrary types of objects
-     * @return Generator|DataObject[]
+     * @return \Generator|DataObject[]
```

Comment:
Removed it and ran linting, and it passed fine - probably because it's in a PHPDoc comment and not in direct code. In direct code, there's a rule that requires adding a `use` statement.

Not sure why this backslash was added. I'll leave it out.